content/applications/sales/crm/acquire_leads/lead_mining.rst

```diff
@@ -1,72 +1,87 @@
 
+Additional filtering options include:
 
-When choosing to target Companied and their contacts you can choose the contacts you are getting based on Role or Seniority. 
-When getting contact information make sure to be aware of the latest EU regulation, get more information about General Data Protection Regulation on
-`Odoo GDPR <http://odoo.com/gdpr>`__, 
+- :guilabel:`Size`: define the range of employees for the companies criteria
+- :guilabel:`Countries`: define leads by the country (or countries) they are located in
+- :guilabel:`States`: further define leads geolocation criteria by state, if applicable
+- :guilabel:`Industries`: define specific scope(s) of industry where leads work in
+- :guilabel:`Sales Team`: choose which Sales Team the leads will be assigned to
+- :guilabel:`Salesperson`: choose which person(s) on the Sales Team the leads will be assigned to
+- :guilabel:`Default Tags`: choose what tags are added directly to the leads once found
 
+Pricing
+=======
 
-You can decide to filter the leads you are getting depending on the size (number of employees) of the companies.
-You can pick the countries your leads are coming from. It is possible to pick multiple countries.
-You can pick the industries your leads are coming from. It is possible to pick multiple industries.
+Lead mining is an In-App Purchase feature and each generated lead costs one credit.
 
-You can choose the salesperson and the Salesteam the lead will be assigned to. 
-You can also add tags to track your generated leads.
+Choosing to receive contact information from each lead will also cost one additional credit.
 
-.. tip::
-    The generated leads will have the name of the company.
+.. note::
+   See here for full pricing information `feature Lead Generation by Odoo IAP
+   <https://iap.odoo.com/iap/in-app-services/167?>`__.
 
-Pricing
-==================
-This is an In-App Purchase feature, each generated lead will cost you one credit.
-If you choose to get contact information each contact will also cost us one additional credit.
-Here is the pricing for this `feature <https://iap.odoo.com/iap/in-app-services/167?>`__, 
-To buy credits you can either go to :menuselection:`CRM --> Configuration --> Settings --> Buy
-Credits`; or go to :menuselection:`Settings --> In-App Purchases --> View my Services`.
-
-.. image:: lead_mining/LM6.png
-   :align: center
+To buy credits, navigate to :menuselection:`CRM --> Configuration --> Settings --> Buy
+Credits`.
 
-.. image:: lead_mining/LM7.png
+.. image:: lead_mining/buy-lead-mining-credits-setting.png
    :align: center
+   :alt: Buy credits from lead mining settings
 
+Alternatively, credits may also be purchased by navigating to :menuselection:`Settings -->
+In-App Purchases --> View my Services`.
 
-.. tip::
-    The blue box will automatically tell you how many credits are going to be consumed.
+.. image:: lead_mining/buy-lead-mining-credits-iap.png
```

Comment:
This image is accurate for 14.0 but inaccurate for 15.0. Let's replace it with the attached image (already compressed), which will be accurate for both.
![view-my-services-setting](https://user-images.githubusercontent.com/101904966/192658541-77949583-cfa6-4688-9521-3855abffa919.png)